### ISSUE TITLE
InvalidArgumentException after composer update to Symfony 3.3.2

### ISSUE THREAD

**Issue by jarekkowol** [Sylius/Sylius#8164]
Any idea to solve this problem?
 
```
(1/1) InvalidArgumentException
Tag "<argument>" with type="service" has no or empty "id" attribute in "../vendor/sylius/sylius/src/Sylius/Bundle/ThemeBundle/DependencyInjection/../Resources/config/services/integrations/assets.xml".
```

**Comment by venyii** [comment]
This was fixed in #8031 (current master)

**Comment by lchrusciel** [comment]
Thanks @venyii 👍
